articles/native-platforms/android/06-rules.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Auth0 Android Quickstarts - Rules
+description: This tutorial will show you how to add customized rules to your app.
+seo_alias: android
+---
+
+## Android - Rules Tutorial
```

Comment:
This title is unnecessary. As far as I can tell, the web page renders the title automatically by combining the platform name and the title you set above, on the yaml part of this document.